ideal-image-slider.js

```diff
@@ -431,6 +431,7 @@ var IdealImageSlider = (function() {
 			beforeChange: function() {},
 			afterChange: function() {}
 		};
+		this.length = 0;
```

Comment:
It allows one to test for the existence of empty Sliders, sliders with no images. Here is a usage example:

```
jQuery(document).ready(function ($) {
	var homeSlider = new IdealImageSlider.Slider({
		selector: '#home-slider',
		height: 'auto',
		effect: 'fade',
		interval: 5000,
		transitionDuration: 600,
		maxHeight: 375
	});

	if( homeSlider.length ) {
		homeSlider.addBulletNav();
		homeSlider.start();
	}
````